cjworkbench/middleware/i18n.py

```diff
@@ -0,0 +1,27 @@
+from django.conf import settings
+
+
+class SetCurrentLocaleMiddleware(object):
+    def __init__(self, get_response):
+        self.get_response = get_response
+        # One-time configuration and initialization.
+
+    def __call__(self, request):
+        # Code to be executed for each request before
+        # the view (and later middleware) are called.
+        locale = None
+
+        if request.user.is_staff:
+            locale = request.GET.get("locale")
+
+        if not (locale in map(lambda p: p[0], settings.LANGUAGES)):
+            locale = settings.LANGUAGE_CODE
+
+        request.currentLocale = locale
```

Comment:
Need to change this var name. Python convention is `snake_case`, not `camelCase`. Furthermore, the word "current" seems redundant.

I think `request.locale_id` is a better name. It follows the lingo in http://unicode.org/reports/tr35/#unicode_locale_id and http://userguide.icu-project.org/locale#examples. And it's better than `request.locale` because `locale` might be an `object`, but `request.locale_id` is obviously a `str`.